SeaPublicWebsite.UnitTests/SeaPublicWebsite.UnitTests.csproj

```diff
@@ -10,6 +10,7 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="CabinetOffice.GovUkDesignSystem" Version="1.0.0-d899eac" />
```

Comment:
Do we really need this here? The tests shouldn't need this library

Reply:
I believe Rider added it for me, I'll remove it